integrations/logging/loki/values.grafana.yaml

```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 ---
 adminUser: admin
-adminPassword: password # notsecret
+adminPassword: password  # notsecret
```

Comment:
Unrelated change

Reply:
This failed the linting because there was not two spaces before the comment.